example/Cargo.toml

```diff
@@ -10,6 +10,10 @@ repository = "https://github.com/alipay/ttrpc-rust"
 homepage = "https://github.com/alipay/ttrpc-rust"
 description = "An example of ttrpc."
 
+[dependencies]
```

Comment:
ok this makes sense, didn't think about it from that perspective. I did it this way since when you import async from ttrpc it fails to build due to async not working with Windows. I tried using conditional dependencies that brought in a feature based on OS but that didn't work due to https://github.com/rust-lang/cargo/issues/1197 (note the nightly changes in cargo did work with https://github.com/rust-lang/cargo/issues/7914 using `rustup override set nightly` & `cargo +nightly build -Z features=itarget`)

I was able to work around this by updating the async marcro to check for linux. Doing that also let me add the examples project to the workspace as well which is a nice improvement for the developer experience in VScode (gives intelisense and compile errors when editing it).